common/script/index.js

```diff
@@ -2338,6 +2338,8 @@ api.wrap = function(user, main) {
           if (analytics != null) {
             analytics.track('acquire item', analyticsData);
           }
+          console.log(user._tmp)
```

Comment:
actually we already have it https://github.com/HabitRPG/habitrpg/blob/develop/tasks/gulp-eslint.js, but it doesn't cover most files because we still have to port them, it'll come with api v3